### ISSUE TITLE
QDM issue with ED/BBD setup

### ISSUE THREAD

**Issue by BrianVEA** [Inve1951/BetterDiscordStuff#48]
Good evening,
I've recently started using Enhanced Discord and been looking at some useful plugins, and this one caught my eye.

Sadly, I've been running in to an issue where if I use this plugin, the function itself works, but it always gives me the confirmation request, and when I either delete, or cancel it, it pulls up once again, even when I click the message just once.

I gave the exact same file to a friend of mine, and he isn't experiencing the same issue.

I'm sorry if I misread something but I wasn't able to find any documentation in regards to my issue.

Thank you, and have a wonderful day.

**--EDIT--**
I just re-installed everything on the canary version of discord and the delete feature seems to be working, just experiencing an issue with the avatar hover plugin, but will try to work that out.

I am not sure if this could be caused by the fact I installed BBD before I installed ED en then uninstalled BBD as I prefered ED over it, I'll try to continue doing some research in to it.

**--EDIT2--**

So, it worked for a while, then I did a restart of the client and somehow it ended up doing the same thing again, I even ended up removing the plugin temporarily and it still continued to have the same issue, so I've got no idea what can be causing it, the issue does how ever stop if I disable the double click edit feature, neither of the two plugins will work, even if the quick delete one is still enabled.

**--EDIT3--** 

And now it works again, I have no idea what is going on with this, sorry for the huge message by now, I am not quite sure what is happening but it seems to be working, will see if it remains stable.

**Comment by Inve1951** [comment]
Well...
The plugins in this repository are designed for BBD only. QDM also pulls [my library](https://github.com/Inve1951/BetterDiscordStuff/blob/master/plugins/0circle.plugin.js) for which the same applies. I have not tested any of my plugins on ED or ED/BBD double setup.

I wasn't even aware that ED could load and run BD plugins but even so I wouldn't be surprised by conflicts or incompatibilities.

QDM has an option to ask for confirmation, by default it does not. It also skips the confirmation when `Shift` is held in addition to `Delete`.

The DoubleClickEdit plugin (if we are talking about the BD one) is no longer maintained and has been superseded by [MessageUtils](https://github.com/mwittrien/BetterDiscordAddons/tree/master/Plugins/MessageUtilities) (which also includes QDM functionality).

**Comment by BrianVEA** [comment]
I appreciate your response.

I do think it appears that I failed to properly use the 0circle plugin before I resolved the issue, I've not had any issue and all your plugins appear to be working for me now, without issue. 

From all the testing I've been doing, all BBD plugins work with ED, as long as you have the BD plugin option enabled in the settings. To me it seems ED is just BBD with some additional functions, and the removal of the BBD emote options.

Sorry for bothering you with this issue but thank you for the reply, and all of your plugins, they're wonderful.
